### ISSUE TITLE
Youtube embedded video volume slider missing, an issue for users with high output volumes

### ISSUE THREAD

**Issue by Fironet** [itchio/itch.io#607]
Is there any way of enabling the volume slider for Youtube embedded videos on itch.io game pages?

I use speakers set to a higher volume to provide headroom so quiet audio on web media can be raised. However due to the embedded videos on itch.io game pages lacking the usual volume slider it means the audio defaults to the last remembered setting (which is frequently loud), leaving only mute as the alternative.

Is this possible with the embedding options or inherent to the narrow embedded video UI Youtube provides?

**Comment by leafo** [comment]
I'm assuming the reason this is happening is because the space where the youtube video is very small. We're going to move the video embed to the center column so it will have more space. I'll close this issue when that happens.

**Comment by Guard13007** [comment]
@leafo That is correct, it auto-hides when below a certain size..unfortunately I do not remember what this size is.
